etna/whatson/models.py

```diff
@@ -829,3 +834,314 @@ class EventFilterForm(forms.Form):
+        verbose_name=_("subtitle"),
+        blank=True,
+        help_text=_("A subtitle for the event."),
+    )
+
+    hero_image = models.ForeignKey(
+        get_image_model_string(),
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    hero_text_colour = models.CharField(
+        max_length=255,
+        verbose_name=_("hero text colour"),
+        help_text=_("The colour of the text in the hero image."),
+        choices=HeroColourChoices.choices,
+    )
+
+    # Key details
+    start_date = models.DateTimeField(
+        verbose_name=_("start date"),
+        null=True,
+    )
+
+    end_date = models.DateTimeField(
+        verbose_name=_("end date"),
+        null=True,
+    )
+
+    min_price = models.IntegerField(
+        verbose_name=_("minimum price"),
+        default=0,
+    )
+
+    max_price = models.IntegerField(
+        verbose_name=_("maximum price"),
+        default=0,
+    )
+
+    dwell_time = models.CharField(
+        max_length=255,
+        verbose_name=_("dwell time"),
+        blank=True,
+        help_text=_("The average dwell time for the exhibition."),
+    )
+
+    target_audience = models.CharField(
+        max_length=255,
+        verbose_name=_("target audience"),
+        blank=True,
+        help_text=_("The target audience for the exhibition."),
+    )
+
+    # Location details
+    location = models.CharField(
+        max_length=255,
+        verbose_name=_("location"),
+        help_text=_("The location of the exhibition venue."),
+    )
+
+    location_url = models.URLField(
+        verbose_name=_("location url"),
+        blank=True,
+        help_text=_("The URL of the exhibition venue."),
+    )
+
+    # Content
+    # TODO: video = . . .
+
+    description = RichTextField(
+        verbose_name=_("description"),
+        help_text=_("A description of the exhibition."),
+        features=settings.EXPANDED_RICH_TEXT_FEATURES,
+    )
+
+    # Need to know
+    need_to_know = RichTextField(
+        verbose_name=_("need to know"),
+        blank=True,
+        help_text=_("Useful information about the exhibition."),
+        features=settings.EXPANDED_RICH_TEXT_FEATURES,
+    )
+
+    need_to_know_cta = models.CharField(
+        max_length=255,
+        verbose_name=_("need to know CTA"),
+        blank=True,
+        help_text=_("The call to action text for the need to know section."),
+    )
+
+    need_to_know_url = models.URLField(
+        verbose_name=_("need to know URL"),
+        blank=True,
+        help_text=_("The URL for the need to know section."),
+    )
+
+    need_to_know_image = models.ForeignKey(
+        get_image_model_string(),
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    # Related content
+    articles_title = models.CharField(
+        max_length=100,
+        blank=True,
+        help_text=_("The title to display for the articles section."),
+    )
```

Comment:
Question: is there a need to limit this to 100 characters? That's verging on the side of restrictively short.